src/7-to-8/major-changes/platforms.rst

```diff
@@ -27,22 +27,11 @@ Platforms
 
    .. code-block:: console
 
-      WARNING - Task <task>: deprecated "host" and "batch system" will be removed at Cylc 9
+      WARNING - Task <task>: deprecated "host" and "batch system" use "platform".
```

Comment:
Is there any way of automating this?

Reply:
Automating what? (Sorry if I'm being dense!)

Reply:
grabbing the actual error message as currently formatted.